www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -120,6 +120,14 @@
         <BlankDesc>Weighted Round Robin</BlankDesc>
         <Required>N</Required>
       </load_balancing_algorithm>
+      <host_port type="IntegerField">
+        <MinimumValue>1</MinimumValue>
+        <Required>N</Required>
+      </host_port>
+      <x_forwarded_host_use_original_host_header type="BooleanField">
```

Comment:
```suggestion
      <x_forwarded_host_verbatim type="BooleanField">
```

small name idea for shortening.. @fabianfrz or “unmodified”

length of previous seems a little over the top

Reply:
"verbatim" sounds more "refined" than "unmodified" (and shorter!). hope @fabianfrz won't mind